16Ianuarie/src/test/java/com/sci/bv/MaximumTest.java

```diff
@@ -0,0 +1,29 @@
+package com.sci.bv;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class MaximumTest {
+
+    /** This is the test for getting the maximum from 3 numbers given by user input
+     *
+     *
+     * @author Marian Pescaru
+     */
+
+    @Test
+    void getMax() {
+        Assertions.assertEquals(Maximum.getMax(5,6),6);
+    }
+
+    @Test
+    void getMaxTrue(){
+        Assertions.assertTrue(Maximum.getMax(2,3)>2);
+    }
+
+    @Test
+    void getMaxFalse(){
+        Assertions.assertFalse(Maximum.getMax(4,5)<5);
```

Comment:
getMaxTrue and getMaxFalse aren't very good tests... you're not checking a single valid response, but that a response is a certain way.

To write more than one test here you could test equal values, test negative values, one negative and one positive...